browser/help/topics/exome-capture-tech.md

```diff
@@ -0,0 +1,11 @@
+---
+id: capture-tech
```

Comment:
```suggestion
id: exome-capture-tech
```
adding just in case (I'm not 100% sure if the name here needs to match up with the the name of the file/what's in the help page table of contents)